### PR TITLE
Create main struture of website

### PR DESCRIPTION
**Pull request by gkaretka**
Signed-off-by: Gregor Karetka karetka.gregor@gmail.com

### CODE REVIEW

**Review by pepol — CHANGES_REQUESTED** [review]
Please remove the non-minified versions of bootstrap files.

File: bootstrap-3.3.7/js/npm.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+// This file is autogenerated via the `commonjs` Grunt task. You can require() this file in a CommonJS environment.
```
Comment (pepol) [inline bootstrap-3.3.7/js/npm.js]: Is this file required? If not, please delete it.

File: css/main.css [hunk 2]
```diff
@@ -0,0 +1,7 @@
+body {
+    background: url("../img/background.jpg");
+}
+
+.navbar-right {
+    margin-right: 2%;
+}
```
Comment (pepol) [inline css/main.css]: This line is missing a newline character at the end. Please add it.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE HTML>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Compotes</title>
+        <script src="js/jquery-3.1.0.min.js"></script>
```
Comment (pepol) [inline index.html]: Wouldn't it work better to just use the absolute addressing: `/js/jquery/...` ?

Also, if it's possible, please delete bootstrap and jquery directories and move the appropriate files into either `js/` or `css/` directories as necessary. Thanks.

File: sk/about.html [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+            <li><a href="../sk/about.html">O nás</a></li>
+        </ul>
+
+        <ul class="nav navbar-nav navbar-right">
+            <li><a href="../en/about.html">EN</a></li>
+        </ul>
+    </nav>
+</div>
+</body>
+</html>
```
Comment (pepol) [inline sk/about.html:26]: Same comments as for `index.html`.

File: en/index.html [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+            <li><a href="../en/about.html">About</a></li>
+        </ul>
+
+        <ul class="nav navbar-nav navbar-right">
+            <li><a href="../index.html">SK</a></li>
+        </ul>
+    </nav>
+</div>
+</body>
+</html>
```
Comment (pepol) [inline en/index.html:26]: Same comments as for `index.html`.

File: en/about.html [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+            <li><a href="../en/about.html">About</a></li>
+        </ul>
+
+        <ul class="nav navbar-nav navbar-right">
+            <li><a href="../sk/about.html">SK</a></li>
+        </ul>
+    </nav>
+</div>
+</body>
+</html>
```
Comment (pepol) [inline en/about.html:26]: Same comments as for `index.html`.

**Review by pepol — CHANGES_REQUESTED** [review]
@gkaretka please change all the links to use the absolute format, if that is possible (please test it and if it's not possible, let me know).

File: bootstrap-3.3.7/js/npm.js (hunk 1 above)
Comment (gkaretka) [reply bootstrap-3.3.7/js/npm.js]: no

**Comment by gkaretka** [comment]
done

**Review by pepol — CHANGES_REQUESTED** [review]
@gkaretka it seems like you (hopefully mistakenly) marked all files as executable (`755` chmod). Please return them to `644` and keep only directories as `755`.

**Comment by gkaretka** [comment]
done

**Review by pepol — APPROVED** [review]

**Comment by pepol** [comment]
@gkaretka Ship it!

**Pull request merged**